StreamingAnalytics/LowLatency/README.md

```diff
@@ -0,0 +1,36 @@
+# Streaming Analytics Low Latency Command and Control Sample
+This example demonstrates the use of Apama to average temperature
+sensor values over a 10 second window. If the average temperature
+exceeds a threshold (80 in this example) then an alert is sent to the
+cloud and a shutdown is initiated.
+
+## Prerequisites
+
+Follow the setup and configuration instructions in the
+[README](../README.md) file in the parent directory before running this sample.
+
+Ensure that the thin-edge device has Python 3 installed, that it is on your `$path`,
+and that you have installed the paho-mqtt Python library:
+
+```
+sudo apt-get install python3
+pip3 install paho-mqtt
+```
+
+## Running the Sample
+
+1. Deploy the Project using `engine_deploy --outputDeployDir project <project-src-dir>`
+(Take reference from [README](../README.md)) and then Copy the project directory to the `/etc/tedge/apama/project`
+directory on the thin-edge device.
+
+2. Restart the Apama service on the thin-edge device with `sudo service apama restart`.
+This restarts the correlator and runs the project that was copied in the previous step.
+
+3. Run the `publisher.py` script from this directory using
+Python 3 with the command `python3 publisher.py`.
+
+4. To see the measurements that will be sent
+to the cloud, run `tedge mqtt sub 'alerts/temperature'`.
+
```

Comment:
Add information about the shutdown operation as well. Like:
To trigger a shutdown operations, project publishes a mqtt message(maybe add a sample message here) to 

>device/operations